src/style.css

```diff
@@ -1,12 +1,63 @@
+@import url('https://fonts.googleapis.com/css2?family=Black+Ops+One&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Concert+One&display=swap');
+
+body {
+  margin: 60px;
+  background: linear-gradient(0deg, #000, #272727);
+  color: #c5c5b7;
+  background-repeat: no-repeat;
+  height: 100vh;
+}
+
 main {
   display: flex;
   justify-content: center;
-  gap: 32px;
+  gap: 62px;
+  font-family: 'Concert One', cursive;
+  flex-wrap: wrap-reverse;
+}
+
+h1 {
+  font-family: 'Black Ops One', cursive;
+  font-size: 48px;
 }
 
 #score {
+  position: relative;
   width: 250px;
   padding: 16px;
+  background: linear-gradient(0deg, #000, #272727);
+}
+
+#score::before,
+#score::after {
+  content: '';
+  position: absolute;
+  left: -2px;
+  top: -2px;
+  background:
+    linear-gradient(
+      45deg,
+      #fb0094,
+      #00f,
+      #0f0,
+      #ff0,
+      #f00,
+      #fb0094,
+      #00f,
+      #0f0,
+      #ff0,
+      #f00
+    );
+  background-size: 400%;
+  width: calc(100% + 4px);
+  height: calc(100% + 4px);
+  z-index: -1;
+  animation: steam 20s linear infinite;
+}
```

Comment:
:fire: :fire: :fire:

Reply:
sorry I can't understand what do you want from me to change